.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
```

Comment:
Okay. Most projects on github add .gitignore - if you still want it to be removed, will remove it.